recipes/ua-nodeset/all/conanfile.py

```diff
@@ -10,34 +12,36 @@ class UaNodeSetConan(ConanFile):
     description = "UANodeSets and other normative files which are released with a specification"
     homepage = "https://github.com/OPCFoundation/UA-Nodeset"
     url = "https://github.com/conan-io/conan-center-index"
-    topics = ("opc-ua-specifications", "uanodeset", "normative-files", "companion-specification")
-
-    no_copy_source = True
+    topics = ("opc-ua-specifications", "uanodeset", "normative-files", "companion-specification", "pre-built")
 
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
+    package_type = "application"
+    settings = "os", "arch", "compiler", "build_type"
 
-    def _extract_license(self):
-        content = tools.load(os.path.join(self.source_folder, self._source_subfolder, "AnsiC", "opcua_clientapi.c"))
-        license_contents = content[2:content.find("*/", 1)]
-        tools.save("LICENSE", license_contents)
+    def layout(self):
+        pass
 
-    def source(self):
-        tools.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
+    def package_id(self):
+        del self.info.settings.compiler
+        del self.info.settings.build_type
```

Comment:
why does it require arch and os, if there is no build and the conandata.yml has no distinction about it?

Reply:
Copied this from the pre-built package template initially and forgot to update it. Thanks for catching this!